manifests/mod/php.pp

```diff
@@ -49,8 +49,13 @@
     $_package_name = undef
   }
 
-  $_lib = "libphp${php_version}.so"
   $_php_major = regsubst($php_version, '^(\d+)\..*$', '\1')
+  $_php_version_no_dot = regsubst($php_version, '\.', '')
```

Comment:
If you had a ```php_scl``` that's actually the SCL name then it'd become:

```puppet
if $apache::php_scl {
  $_lib = "lib${apache::php_scl}-${mod}.so"
} else {
  $_lib = "lib${mod}.so"
}
```

Note the above assumes you change params to inherit from globals.